src/core/public/mocks.ts

```diff
@@ -83,6 +83,7 @@ function createCoreSetupMock({
     uiSettings: uiSettingsServiceMock.createSetupContract(),
     injectedMetadata: {
       getInjectedVar: injectedMetadataServiceMock.createSetupContract().getInjectedVar,
+      getBranding: injectedMetadataServiceMock.createSetupContract().getBranding,
```

Comment:
This was a missed addition when we added branding - I fixed it as part of actually relying on these mocks from porting discover tests.

Reply:
Thank you!